clients/web/src/views/layout/ResetPasswordView.js

```diff
@@ -38,14 +38,6 @@ var ResetPasswordView = View.extend({
 
             this.$('#g-reset-password-button').girderEnable(false);
             this.$('.g-validation-failed-message').text('');
-        },
-
-        'click a.g-register-link': function () {
```

Comment:
I don't think you want to be removing these, since you're not updating the template for *this* particular view.